examples/node/MarketDataSubscription_LongPoll.js

```diff
@@ -0,0 +1,65 @@
+var fs = require('fs');
+var request = require("request");
+
+var host = process.argv[2] || 'http-api.openbloomberg.com';
+var port = 3000;
+var url = 'https://' + host + ':' + port;
+var NUM_REQUEST = 10;   // Number of long-poll requests before unsubscribe
+var counter = 0;
+var opt = {
+    cert: fs.readFileSync('client.crt'),
+    key: fs.readFileSync('client.key'),
+    ca: fs.readFileSync('bloomberg.crt')
+};
+
+function subscribe()
+{
+    request.post({
+        url: url + '/subscription?action=start',
+        body: [
+                { security: 'AAPL US Equity', correlationId: 0, fields: ['LAST_PRICE'] },
+                { security: 'GOOG US Equity', correlationId: 1, fields: ['LAST_PRICE'] }
+              ],
+        json: true,
+        agentOptions: opt
+    }, function (error, response, body) {
```

Comment:
What happens when `error` occurs?  Should we just log and bail?  Same comment for the other callback handlers in this file.